app/views/admin/categories/new.html.erb

```diff
@@ -0,0 +1,18 @@
+<h4 class="text-center">Category Registration</h4>
+<%#= form_for @category do |f| %>
+<%= form_for(@category, :url => {:controller => :categories, :action => :create}) do |f| %>
```

Comment:
you can remove the :url values here..
Rails automatically knows that new.html.erb will pass data to the create method, so need to put the url...

You can refactor it to ->
<%= form_for([:admin, @category]) do |f| %>